packages/spec/src/spec.emu.html

```diff
@@ -287,7 +287,7 @@ <h1>Syntactic Grammar</h1>
     DecoratorList? `model` Identifier TemplateParameters? ModelHeritage? `{` ModelBody? `}`
 
 IsModelHeritage :
-    `is` ReferenceExpression
+    `is` ReferenceExpression`[]`?
```

Comment:
nit:

```suggestion
    `is` ReferenceExpression `[]`?
```

Reply:
Actually, I think this is not the same as what's implemented. This allows a single level of `[]` but it looks like the implementation allows `model P is Q[][][]`?


I guess that could be expressed like this if again we choose to avoid naming ReferenceExpressionOrReferenceExpressionArray:

```
IsModelHeritage:
   `is` ReferenceExpression
   IsModelHeritage `[]`
```

But it does obscure the meaning a bit.

I'm also starting to wonder if we wouldn't get a better error recovery experience if we parsed any expression and left it to checker to say this expression can't be `is` base. But we can leave  leave that for future, and maybe look at it all up for similar cases that might benefit.

Reply:
Offline, discovered that this would work:

```
alias Y = A | B
model F is Y[];
```

But not this:
```
model F is (A | B)[];
```

That suggests that maybe we really should parse `is Expression` and just check if the expression results in something is-able as a semantic check, not a parsing thing.